content/blog/2021/announcing-in-kind-sponsorship.md

```diff
@@ -0,0 +1,42 @@
+Title:Call for In-Kind Sponsors for PyCon India 2021
+Date: 2021-06-06 17:00
+Category: 2021
+Slug: call-for-in-kind-sponsors
+Summary: Want to give back to the Python community and have more reach among the community members? Here's your chance to put your organization in the limelight of the Python community by sponsoring the largest Python conference in India.
+
+
+As you all might know that [PyCon India 2021](https://in.pycon.org/2021/) is happening online and we are not taking sponsorship for this year's event and all logistical expenses are being managed through the ticket amount collection itself. Ticket prices are also kept very minimal so that everyone can afford them. However, we are starting with In-kind Sponsorship this year so that Companies/Organizations who are still interested to contribute to the event and add value to the community can come forward and make the most out of this opportunity.
```

Comment:
```suggestion
As you all might know that [PyCon India 2021](https://in.pycon.org/2021/) is happening online and we are not taking sponsorship for this year's event and all logistical expenses are being managed through the ticket amount collection itself. Ticket prices for the conference have been kept very minimal so everyone could afford them. Henceforth, this year we initiated In-kind Sponsorships at PyCon India. Companies and organizations with an interest to contribute and add value to event and the community can come forward to make the most out of this opportunity.
```

Better alternatives/suggestions for initiating needed.

Reply:
what about 'started'?